context.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/spf13/viper"
+	yaml "gopkg.in/yaml.v2"
+
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/fnproject/cli/config"
+	"github.com/olekukonko/tablewriter"
+	"github.com/urfave/cli"
+)
+
+var contextsPath = filepath.Join(config.GetHomeDir(), config.RootConfigPathName, config.ContextsPathName)
+var defaultContextPath string
+
+func contextCmd() cli.Command {
+	return cli.Command{
+		Name:  "context",
+		Usage: "manage context",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Aliases:   []string{"c"},
+				Usage:     "create a new context",
+				ArgsUsage: "<context>",
+				Action:    create,
+				Flags: []cli.Flag{
+					cli.StringSliceFlag{
+						Name:  "config",
+						Usage: "context configuration",
+					},
+					cli.StringFlag{
```

Comment:
Yes for sure, the next step would be when the provider defined was say 'oracle' we would create a context file with specific oracle fields. This will lead on from the work @mantree has just done.